### PR TITLE
upgraded to new docs theme

### PR DESCRIPTION
**Pull request by alexwolfe**
## Description
I updated the docs templates to match with the upgraded Teleport docs template. This will also fix many of our SEO issues including js/css that was not minified. 

## Type of change
This is mostly a cosmetic change but will have a impact on SEO

### CODE REVIEW

File: docs/theme/toc.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h3 class="side-nav-section-header">
                   <a class="{% if nav_item.active%}is-active{%endif%}" href="{{ nav_item.url|url }}">{{ nav_item.title }}</a>
                 </li>
               </ul>
-              {% if nav_item.active%}
+              <!-- {% if nav_item.active%}
```
Comment (a-palchikov) [inline docs/theme/toc.html:20]: if this is not needed, I'd rather remove it than comment it.

**Review by a-palchikov — APPROVED** [review]
hard to validate the change - if things are visual, would be helpful to see the differences but otherwise - lgtm

File: docs/theme/toc.html (hunk 1 above)
Comment (alexwolfe) [reply docs/theme/toc.html:20]: I'm keeping this around as a reference when we expand the navigation again. I'll remove all old stuff when we upgrade to the new design.

**Review by r0mant — APPROVED** [review]

**Review by benarent — APPROVED** [review]
Tested locally and everything looks good.

**Pull request merged**